analytics/src/analytics/datasets/utils.py

```diff
@@ -67,9 +67,9 @@ def load_json_data_as_df_from_object(
     """
     # flatten the nested json into a dataframe
     df = pd.json_normalize(json_data)
-    # reorder and rename the columns
-    df = df[column_map.keys()]
-    df = df.rename(columns=column_map)
```

Comment:
I'm not sure I love the idea of us removing this step from a general utility function, especially if the parameter for `column_map` still says it will rename them.

We should either change the function signature and docstrings, or restore the rename step.

Reply:
And if we're going to change the function signature then we should also remove this [`COLUMN_MAP`](https://github.com/HHS/simpler-grants-gov/blob/6416b063a10fecbff4a067eaa92d22bd01fec2c5/analytics/src/analytics/datasets/etl_dataset.py#L35-L63) class global so we don't have folks thinking the columns are getting renamed.